test/TestLinearPremiumPriceOracle.js

```diff
@@ -0,0 +1,72 @@
+const ENS = artifacts.require('@ensdomains/ens/ENSRegistry');
+const BaseRegistrar = artifacts.require('./BaseRegistrarImplementation');
+const DummyOracle = artifacts.require('./DummyOracle');
+const LinearPremiumPriceOracle = artifacts.require('./LinearPremiumPriceOracle');
+
+const namehash = require('eth-ens-namehash');
+const sha3 = require('web3-utils').sha3;
+const toBN = require('web3-utils').toBN;
+
+const DAY = 86400;
+
+contract('LinearPremiumPriceOracle', function (accounts) {
+    let priceOracle;
+
+    before(async () => {
```

Comment:
ditto

Reply:
None of the tests here change the state, so I think changing this one to `beforeEach` would only slow the test down.